docs/docs-beta/docs/dagster-plus/deployment/serverless/run-isolation.md

```diff
@@ -1,8 +1,73 @@
 ---
-title: "Serverless run isolation"
-displayed_sidebar: "dagsterPlus"
-sidebar_label: "Run isolation"
+title: 'Serverless run isolation'
+displayed_sidebar: 'dagsterPlus'
+sidebar_label: 'Run isolation'
 sidebar_position: 40
 ---
 
-# Serverless run isolation
\ No newline at end of file
+# Serverless run isolation
+
+Dagster+ Serverless offers two settings for run isolation: isolated and non-isolated. Isolated runs are the default and are intended for production, while non-isolated runs are intended for faster iteration during development.
```

Comment:
Dagster+ Serverless allows users to select isolated or non-isolated runs for a deployment. Isolated runs are the default and are intended for production, while non-isolated runs are intended for faster iteration during development.